ui/ducks/send/send.js

```diff
@@ -1287,6 +1292,29 @@ const slice = createSlice({
           }):
           draftTransaction.amount.error = INSUFFICIENT_TOKENS_ERROR;
           break;
+        // set error to INSUFFICIENT_TOKENS_ERROR if the token balance is lower
+        // than the amount of token the user is attempting to send.
+        case draftTransaction.asset.type === AssetType.NFT &&
+          draftTransaction.asset.details.standard === TokenStandard.ERC1155 &&
+          !isERC1155BalanceSufficient({
+            tokenBalance: draftTransaction.asset.details.balance ?? '0x0',
+            amount: draftTransaction.amount.value,
+          }):
+          draftTransaction.amount.error = INSUFFICIENT_FUNDS_ERROR;
+          break;
+        // if the amount of tokens is negative or equal to zero, set error to NEGATIVE_OR_ZERO_AMOUNT_TOKENS_ERROR
+        case amountValue.lessThanOrEqualTo() &&
```

Comment:
Is an argument supposed to be passed here?  I'm wondering why it's not `lessThanOrEqualTo(0)`